src/webui/www/private/views/rss.html

```diff
@@ -406,6 +442,12 @@
             if (path === "")
                 visibleArticles = visibleArticles.filter((a) => !a.isRead);
 
+            const rssFilterInput = document.getElementById("rssFilterInput");
+            if (rssFilterInput.value) {
```

Comment:
Don't rely on implicit conversion to bool. `rssFilterInput.value !== undefined` or something like that.